### PR TITLE
WT-5083 Add mips64el support

### PR DESCRIPTION
**Pull request by lina-not-linus**

### CODE REVIEW

**Comment by wtbuild** [comment]
autobuild: Can one of the admins verify this patch?

**Comment by tetsuo-cpp** [comment]
autobuild: retest this please

**Comment by tetsuo-cpp** [comment]
autobuild: retest this please

File: SConstruct [hunk 1]
```diff
@@ -244,6 +244,7 @@ condition_map = {
     'POWERPC_HOST' : False,
     'WINDOWS_HOST' : env['PLATFORM'] == 'win32',
     'X86_HOST' : True,
+    'MIPS64EL_HOST' : True,
```
Comment (tetsuo-cpp) [inline SConstruct]: Is it necessary to set this to `True`? Our automated testing couldn't compile on Windows and I suspect that this is why.

**Comment by lina-not-linus** [comment]
I'm sorry that my mistake caused everyone trouble. I repulled the develop branch and recompiled the validation on mips64el, which is correct. thank you     [#4852](https://github.com/wiredtiger/wiredtiger/pull/4852)

File: SConstruct (hunk 1 above)
Comment (lina-not-linus) [reply SConstruct]: Sorry, not here

**Comment by tetsuo-cpp** [comment]
Closing this. Superseded by #4852.

**Comment by Ccaomei** [comment]
> I'm sorry that my mistake caused everyone trouble. I repulled the develop branch and recompiled the validation on mips64el, which is correct. thank you [#4852](https://github.com/wiredtiger/wiredtiger/pull/4852)



> Closing this. Superseded by #4852.

1. 

- 

> I'm sorry that my mistake caused everyone trouble. I repulled the develop branch and recompiled the validation on mips64el, which is correct. thank you [#4852](https://github.com/wiredtiger/wiredtiger/pull/4852)

您好，我按照您说的，在mips64el上操作了一下，编译mongo源码的时候，提示mongo-r3.2.22/src/third_party/wiredtiger/src/support/global.c:58: undefined reference to `__wt_checksum_init'。 这个是在链接生成mongod的时候出现的错误

**Pull request closed without merging**